new/PRS14512Pistis.xml

```diff
@@ -0,0 +1,61 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS14512Pistis" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Ṗisṭis</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="JK"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="JK" when="2024-07-24">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="2">
+                    <persName xml:lang="gez" xml:id="n1">ጲስጢስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Ṗisṭis</persName>
+                    <faith type=""/>
+                    <birth/>
+                    <death/>
```

Comment:
I personally remove all empty strings (i.e. <birth/>, <death/> and <faith type=""/>) because if they remain, they will attract unnecessary attention from the user.

Reply:
I have not come across such a case of very many community members as owners so far. What I have seen are for example protective scrolls in which an owner is named, who desired protection for himself as well as to some members of his family. In such cases, I encoded the first person as the owner and the other one or two persons as attestations. To my opinion it is helpful to encode a set of persons, which are related to each other. I think, this could be helpful to identify some of them, who are mentioned together as well in other places. However, this is also possible, when the names are listed in history without markup, as Denis suggested.

Reply:
If the persons are related, sure, and this is most frequently indicated (husband, wife, and sons). Larger groups of people appear in more recent books, as the ways of book aquisition modifies: not only individuals and families, but communities commission / buy books... Then many people can be mentioned.

Reply:
In this case, there are numerous layers of erasure in most of the supplications at the end of texts or text parts. I think I have gone through the manuscript quite carefully (although, of course, maybe I missed some), and there are some names that occur frequently, while others appear only one or two times.

Now I have created files for them, so in this case I will mention them all. I think your reasoning about community members sounds reasonable, so I will encode the main names as owners and the others as "hasAttestation".

Thank you both!